package.json

```diff
@@ -75,7 +75,7 @@
     "protractor": "^4.0.12",
     "systemjs": "^0.19.41",
     "tslint": "^4.2.0",
-    "typescript": "^2.0.10",
+    "typescript": "~2.3.4",
```

Comment:
Why was the dependency changed from minor releases to patch releases?
Should have been `^2.3.4`

Reply:
At the moment, protractor types definition have a little bug, so tests are failed with typescript 2.4
https://github.com/mgechev/angular-seed/issues/2013